react-native-git-upgrade/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "react-native-git-upgrade",
+  "version": "0.1.0",
+  "license": "BSD-3-Clause",
+  "description": "The React Native upgrader tool",
+  "main": "cli.js",
+  "bin": {
+    "react-native-git-upgrade": "index.js"
+  },
+  "engines": {
+    "node": ">=4"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/react-native.git"
+  },
+  "scripts": {
+    "test": "mocha"
```

Comment:
Are there any tests? If not leave this out. If there will be tests they should be using Jest as the code will be merged to the RN CLI which uses Jest for testing.